src/device_imxrt.c

```diff
@@ -0,0 +1,278 @@
+#include "device.h"
+#include "log.h"
+
+#if MG_DEVICE == MG_DEVICE_RT1020 || MG_DEVICE == MG_DEVICE_RT1060
+
+#include <flexspi.h>  /**************************************************************************************************************************/
+// identical for both 1020 and 1060, they have the same controller. This should be part of Mongoose, not read from the example
+
+
+// these are defined in the device CMSIS .h file, so examples will have their own, we should use a different name or #undef
+// hence, the example flexspi.h does not have these macros, that is NXP code carrying their license
+// Mongoose does not include any CMSIS file but these would go into mongoose.h and the user including mongoose.h and the device .h both definitions would clash
```

Comment:
This is to meditate on...
@cpq ?

Reply:
Ditto - I do not feel easy about that include.